tests/test_integration.py

```diff
@@ -1,2 +1,10 @@
+from pathlib import Path
+
+TEST_PROJECTS = [str(p.stem) for p in (Path(__file__).parent / "contracts").iterdir()]
+
+
 def test_integration(project):
-    assert "solcontract" in project.contracts
+    for proj in TEST_PROJECTS:
```

Comment:
If you run this test when using the old implementation, it fails (because source ID is always the same for each contract)